Samples/src/FSharpWithQSharp/README.md

```diff
@@ -0,0 +1,41 @@
+# Using Q# with F#
+
+This sample shows how to use Q# code with F# classical driver. 
+
+Q# code is compiled to C# for the purposes of local simulation, so using Q# with C# is very straightforward, as described in the documentation. Using Q# with F# is a little less obvious, since you can't mix Q# and F# files in the same project. You have to create a Q# library (which will yield a .csproj project with only Q# files in it) and to reference it from a purely F# application.
+
+The steps are as follows:
+
+1. Create a Q# library `QuantumCode` and write your quantum code in it.
+2. Create an F# application (in this case a console app targeting .NET Core) `FsharpDriver`.
+3. Add a reference to the Q# library to the F# application.
+
+   You can use [Reference Manager](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-add-or-remove-references-by-using-the-reference-manager) in Visual Studio to do that, or you can add the reference from the command line:
```

Comment:
The docs team tends to suggest removing the `en-us` from URLs so that they automatically redirect to user's preferred languages when available.

```suggestion
   You can use [Reference Manager](https://docs.microsoft.com/visualstudio/ide/how-to-add-or-remove-references-by-using-the-reference-manager) in Visual Studio to do that, or you can add the reference from the command line:
```